<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Press Start 2P</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pixel Spirit</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/UI/icon.png</normaloff>
    <normalon>../assets/UI/icon.png</normalon>../assets/UI/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/UI/resources.qrc">:/background.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Welcome">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>420</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/UI/resources.qrc">:/MainWindow/main-page-welcome.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>324</y>
      <width>776</width>
      <height>72</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/UI/resources.qrc">:/MainWindow/title.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>520</y>
      <width>484</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="icon">
     <iconset resource="../assets/UI/resources.qrc">
      <normaloff>:/MainWindow/spirit-house-button.png</normaloff>
      <normalon>:/MainWindow/spirit-house-button.png</normalon>:/MainWindow/spirit-house-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>484</width>
      <height>71</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="TopBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/UI/resources.qrc">:/topBar.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/UI/resources.qrc"/>
 </resources>
 <connections/>
</ui>
